carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/Messager.java

```diff
@@ -31,49 +26,28 @@
  * @author akhursevich
  */
 
-public enum Messager {
-    TEST_STARTED(
-            "\r\n" +
-                    "======================================================================================================================================\r\n"
-                    +
-                    "INFO:%s TEST [%s] STARTED at [%s]"),
+// TODO: move to messager package
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1135)